common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1774,7 +1779,23 @@ object CelebornConf extends Logging {
       .categories("ha")
       .version("0.3.0")
       .timeConf(TimeUnit.SECONDS)
-      .createWithDefaultString("5s")
+      .createWithDefaultString("45s")
+
+  val HA_MASTER_RATIS_CLIENT_RPC_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.master.ha.ratis.raft.client.rpc.timeout")
+      .internal
+      .categories("ha")
+      .version("0.3.2")
+      .timeConf(TimeUnit.SECONDS)
+      .createWithDefaultString("30s")
+
+  val HA_MASTER_RATIS_CLIENT_RPC_WATCH_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.master.ha.ratis.raft.client.rpc.watch.timeout")
+      .internal
+      .categories("ha")
+      .version("0.3.2")
+      .timeConf(TimeUnit.SECONDS)
+      .createWithDefaultString("45s")
```

Comment:
Looks larger than the original default value.

ratis client default config
```properties
raft.client.rpc.request.timeout=3000ms
raft.client.rpc..watch.request.timeout=10000ms
```

https://github.com/apache/ratis/blob/2e4e75bdc9671d9ac03eae50ce02ff4aa48ced1f/ratis-client/src/main/java/org/apache/ratis/client/RaftClientConfigKeys.java#L44-L56

Reply:
Yes, setting this config larger than the default value is on purpose.

Reply:
OK, docs will be updated in this PR.